File: 088/merge-sorted-array.js
/**
 * @param {number[]} nums1
 * @param {number} m
 * @param {number[]} nums2
 * @param {number} n
 * @return {void} Do not return anything, modify nums1 in-place instead.
 */
var merge = function (nums1, m, nums2, n) {
    nums1.length = m;
    nums2.length = n;
    Array.prototype.push.apply(nums1, nums2);
    nums1.sort((a, b) => a - b);
};

File: 088/merge-sorted-array.py
class MergeSortedArray:
    def merge(self, nums1, m, nums2, n):
        """
        :type nums1: List[int]
        :type m: int
        :type nums2: List[int]
        :type n: int
        :rtype: void Do not return anything, modify nums1 in-place instead.
        """
        if n != 0:
            nums1[m:m + n] = nums2
            nums1[0:m + n] = sorted(nums1[0:m + n])
            # print(nums1)